ophyd/utils/epics_pvs.py

```diff
@@ -305,7 +309,10 @@ def _wait_for_value(signal, val, poll_time=0.01, timeout=10, rtol=None, atol=Non
         ttime.sleep(poll_time)
         if poll_time < 0.1:
             poll_time *= 2  # logarithmic back-off
-        current_value = signal.get()
+        if is_array:
+            current_value = signal.get(count=len(val))
+        else:
+            current_value = signal.get()
```

Comment:
```suggestion
        current_value = signal.get(**get_kwargs)
```

minor style suggestion (across 3 comments)

Reply:
op cit